common/reviews/api/gulp-core-build.api.ts

```diff
@@ -185,9 +185,28 @@ interface IExecutable {
 }
 
 // @public
+interface IJestConfig {
+  ci?: boolean;
```

Comment:
>ci [](start = 2, length = 2)

Choose a more informative name using common English words.

For example, the new API Extractor task has a boolean "localBuild" which is the opposite of "--ship"

Reply:
Why are all of these optional? They should just be passed in using Partial<IJestConfig>

---
In reply to: [145562646](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145562646) [](ancestors = 145562646)

Reply:
How is Partial<IJestConfig>better?

---
In reply to: [145567629](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145567629) [](ancestors = 145567629,145562646)

Reply:
Removed ci parameter.

---
In reply to: [145562646](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145562646) [](ancestors = 145562646)